src/re_console/core.cljs

```diff
@@ -43,7 +45,7 @@
         [:div.re-console-container
          {:on-click #(dispatch [:focus-console-editor console-key])}
          [:div.re-console
-          [console-items console-key @items]
+          [console-items console-key @items (:to-str-fn eval-opts)]
```

Comment:
I like it! Just add it in the doc and in `cljs-repl-web` too and we are on! :dancers: